modules/react/button/lib/PrimaryButton.tsx

```diff
@@ -106,12 +103,10 @@ export interface PrimaryButtonProps extends Themeable, GrowthBehavior, BaseButto
   /**
    * Button icon positions can either be `start` or `end`.
    * If no value is provided, it defaults to `start`.
-   * @default 'start'
    */
   iconPosition?: IconPositions;
   /**
    * If set to `true`, transform the icon's x-axis to mirror the graphic
-   * @default false
```

Comment:
I see a number of places where `@default` has been removed from the JSDocs due to smarter parsing mentioned [here](https://github.com/Workday/canvas-kit/pull/1962#discussion_r1117637643).

However, in this case, it isn't showing a default of `false` for `shouldMirrorIcon` in the prop table despite it being set in PrimaryButton's `createComponent`:

<img width="1047" alt="image" src="https://user-images.githubusercontent.com/146020/221317202-a48d7a77-e1b7-446e-94b1-1a5404f87e47.png">

Also, I noticed SecondaryButton (which wasn't touched in this PR and still has `@default false` for `shouldMirrorIcon`) incorrectly shows a default of `true` in the prop table:

<img width="1036" alt="Screenshot 2023-02-24 at 3 06 09 PM" src="https://user-images.githubusercontent.com/146020/221317660-a670c7b1-d38e-4d2e-9dd7-dd1bd142f3ac.png">